twake/backend/node/src/services/workspaces/web/controllers/workspaces.ts

```diff
@@ -106,9 +106,17 @@ export class WorkspacesCrudController
 
     const allCompanyWorkspaces = await this.workspaceService.getAllForCompany(context.company_id);
 
+    const companyUser = await this.companyService.getCompanyUser(
+      { id: context.company_id },
+      { id: context.user.id },
+    );
+
     const allUserWorkspaceRolesMap = await this.workspaceService
       .getAllForUser({ userId: context.user.id }, { id: context.company_id })
-      .then(uws => new Map(uws.map(uw => [uw.workspaceId, uw.role])));
+      .then(
+        uws =>
+          new Map(uws.map(uw => [uw.workspaceId, companyUser.role == "admin" ? "admin" : uw.role])),
```

Comment:
company "owner" too is admin of the workspace

Reply:
Now you have utils functions to get if this is a "at least" admin level. Search for `hasCompanyAdminLevel` in project

Reply:
fixed